mcs/class/aot-compiler/Makefile

```diff
@@ -97,7 +118,13 @@ endif
 
 ifeq ($(PROFILE), $(DEFAULT_PROFILE))
 
-IMAGES = $(mscorlib_aot_image) $(mcs_aot_image) $(CSC_IMAGES)
+ifndef DISABLE_BUNDLE_CORLIB
+CORLIB_IMAGE = $(runtime_with_mscorlib)
```

Comment:
This is confusing to read, just use `IMAGE` and change the assignment below to IMAGES += $(mcs_aot_image) $(CSC_IMAGES)

Reply:
Will do